week-2-practise/level-3/2-validation.js

```diff
@@ -6,7 +6,21 @@
 
 var excludedNums = [6, 14, 91, 111];
 
-function validate(num) {}
+function validate(num) {
+  return isInt(num) && isNotEqualToExecludedNumsArray(num);
+}
+
+function isInt(number) {
+  return (
+    typeof number === "number" &&
+    isFinite(number) &&
+    Math.floor(number) === number
+  );
+}
+
+function isNotEqualToExecludedNumsArray(number) {
```

Comment:
Also, try to be careful with spelling. You wrote excluded correctly on the next line!